inspirehep/dojson/hep/fields/bd70x75x.py

```diff
@@ -25,29 +25,25 @@
 from __future__ import absolute_import, division, print_function
 
 from dojson import utils
+import re
 
 from inspirehep.utils.helpers import force_force_list
-from inspirehep.utils.record import get_value
 
 from ..model import hep, hep2marc
 from ...utils import get_record_ref
 
+ORCID = re.compile('\d{4}-\d{4}-\d{4}-\d{3}[0-9Xx]')
+
 
 @hep.over('authors', '^701..')
```

Comment:
As we discussed with @jacquerie some messy rules, like that one ( and 700,100,701), will be centralized in a file/rule after moving to the newest schemas(v15.0).